src/bootstrap/test.rs

```diff
@@ -925,6 +925,9 @@ impl Step for RustdocGUI {
                     .env("RUSTDOC", builder.rustdoc(self.compiler))
                     .env("RUSTC", builder.rustc(self.compiler))
                     .current_dir(path);
+                if entry.file_name() == "test_docs" {
+                    cargo.env("RUSTDOCFLAGS", "-Zunstable-options --generate-link-to-definition");
+                }
```

Comment:
Could you instead add a new test called something like `link_to_definition` and enable `--generate-link-to-definition` only for that? Enabling `--generate-link-to-definition` for `test_docs` in particular is surprising because there's nothing in the name that would make you think that.

Reply:
Ok!